server/controllers/usersController.api.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const { User } = require('../models');
+
+exports.getUser = (req,res) => {
+    const { id } = req.body;
+    User.findOne({
+        id
+    }, (err, user) => {
+        if (err) {
+            return res.status(500).json({ message: 'an error occurred' });
+        } else if (user) {
+            const { id, name, email } = user;
+            return res.json({
+                status: 200,
+                id,
+                name,
+                email
+            });
+        } else {
+            return res.status(500).json({ message: 'an error occurred' });
```

Comment:
I think .exec() would be better since it could handle the last else? So it would just be like  .exec((err, user) => { 
if (err) do error stuff; 
return res.json( stuff you want to return ) }
unless you don't need this 3rd else since findOne will return the user or an error? Not sure though.

Reply:
Can I just get rid of this and use your userById route?

Reply:
It wouldn't be the userById route since it's not an actual route but it's middleware. But the code for getting a user would be the same thing; something like: 
    User.findById(id).exec((err, user) => {
        if (err || !user) {
            return res.status(400).json({
                error: 'User not found'
            });
        }
        const { _id, name, email } = user;
        return res.status(200).json(user: { _id, name, email });
    });